lib/fs.js

```diff
@@ -411,13 +411,7 @@ fs.accessSync = function(path, mode) {
 };
 
 fs.exists = function(path, callback) {
-  if (handleError((path = getPathFromURL(path)), cb))
-    return;
-  validatePath(path);
-  if (!nullCheck(path, cb)) return;
-  var req = new FSReqWrap();
-  req.oncomplete = cb;
-  binding.stat(pathModule.toNamespacedPath(path), req);
+  fs.access(path, fs.F_OK, cb);
```

Comment:
Is the behavior of `fs.exists` altered if callback is not a function? It looks like `fs.access` throws a TypeError in that case.

Reply:
`cb` is a function (it's defined on the line below) and it checks that `callback` is truthy.